docs/user/openstack/ovs-dpdk.md

```diff
@@ -156,93 +156,91 @@ spec:
     enabled: false
 ```
 
-### Enable `vfio-noiommu` driver
+### Label your CNF workers
 
-In the case of deploying workers in virtual machines, the underlying virtualization platform doesn't support
-a virtualized IOMMU and we need to explicitly enable it via `options vfio enable_unsafe_noiommu_mode=1`.
-This can be done via a MachineConfig resource:
+Once the CNF workers are deployed, you must label them SR-IOV capable:
 
-``` yaml
-kind: MachineConfig
-apiVersion: machineconfiguration.openshift.io/v1
-metadata:
-  name: 99-vfio-noiommu 
-  labels:
-    machineconfiguration.openshift.io/role: worker
-spec:
-  config:
-    ignition:
-      version: 3.2.0
-    storage:
-      files:
-      - path: /etc/modprobe.d/vfio-noiommu.conf
-        mode: 0644
-        contents:
-          source: data:;base64,b3B0aW9ucyB2ZmlvIGVuYWJsZV91bnNhZmVfbm9pb21tdV9tb2RlPTEK
+```sh
+oc label node <node-name> feature.node.kubernetes.io/network-sriov.capable="true"
 ```
 
-More documentation about that can be found [here][3].
+These workers will be used to run the CNF workloads.
 
-### Bind the `vfio-pci` kernel driver to the appropriate NICs
+### Install the SRIOV Network Operator and configure a network device
 
-The workers connected to the `VFIO_NETWORK` require the `vfi-pci` kernel driver to be bound to the ports attached to the `VFIO_NETWORK`.
+You must install the SR-IOV Network Operator. To install the Operator, you will need access to an account on your OpenShift cluster that has `cluster-admin` privileges. After you log in to the account, [install the Operator][3].
 
-We basically need to create a MachineSet for the workers which will be attached to the `VFIO_NETWORK` network.
-The MachineConfig used in the MachineSet should contain the necessary SystemD unit definitions to install the poll mode driver (`vfio-pci`) on boot.
-This [Ansible role][4] provides an easy way to generate the required MachineSet.
-The playbooks also provide the ability to automatically install the MachineSet, but it is highly recommended to use the playbooks for generating a MachineSet file (.yaml).
-Then the user should explicitly install this on their cluster using the appropriate `oc` commands.
+Then, [configure your SR-IOV network device][4]. Note that only `netFilter` needs to be used from the `nicSelector`, as we'll give the Neutron network ID used for DPDK traffic.
 
-To generate the MachineSet file, run the following commands:
+Example of `SriovNetworkNodePolicy` named `dpdk1`:
 
-``` bash
-openstack network show <name of VFIO network> -f value -c id
-ansible-playbook play.yaml -e network_ids="ID of VFIO network" --extra-vars "mc_config_file=/tmp/vhostuser.yaml"
-oc create -f /tmp/vhostuser.yaml
+```
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: dpdk1
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: vfio-pci
```

Comment:
should we mention the `isRdma` here?

Reply:
it's mentioned L193

Reply:
i've added it, it was a good idea. Thanks